lib/ansible/modules/database/postgresql/postgresql_lang.py

```diff
@@ -214,24 +241,46 @@ def main():
     force_trust = module.params["force_trust"]
     cascade = module.params["cascade"]
     fail_on_drop = module.params["fail_on_drop"]
+    sslrootcert = module.params["sslrootcert"]
 
     if not postgresqldb_found:
         module.fail_json(msg="the python psycopg2 module is required")
 
+    # To use defaults values, keyword arguments must be absent, so
+    # check which values are empty and don't include in the **kw
+    # dictionary
     params_map = {
         "login_host": "host",
         "login_user": "user",
         "login_password": "password",
         "port": "port",
-        "db": "database"
+        "db": "database",
+        "ssl_mode": "sslmode",
+        "ssl_rootcert": "sslrootcert"
```

Comment:
The param mapping isn't correct, please, change it to:
```
244     sslrootcert = module.params["ssl_rootcert"]
...
252     params_map = {
253         "login_host": "host",
254         "login_user": "user",
255         "login_password": "password",
256         "port": "port",
257         "db": "database",
258         "ssl_mode": "sslmode",
259         "sslrootcert": "ssl_rootcert"
260     }
```
It works if it's changed so.
Probably we should check all of other postgresql* modules with supporting sslrootcert.
Please, change it and check your example again. It needs that somebody else except me check it.
Thank you!

Reply:
I think you are right, I will update this PR now.

Reply:
I think we need to change line 258 too.

Reply:
Actually no scratch that I think the problem is actually only line 244. The `params_map` is fine since it's keys are the module params and the values are the psql key words which don't contain a `_` (see http://initd.org/psycopg/docs/module.html and  https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-PARAMKEYWORDS)

Reply:
You are perfectly right. Moreover, I've changed line 258 during the previous test too, otherwise it couldn't work )
I cloned your repo and test it. Seems it works correctly.
```
(0, '\r\n{"lang": "plpythonu", "invocation": {"module_args": {"lang": "plpythonu", "ssl_rootcert": "/root/root.crt", "ssl_mode": "verify-full", "login_unix_socket": "", "login_user": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER", "login_host": "myhost", "db": "ep2_alfa_bank", "force_trust": false, "state": "present", "cascade": false, "login_password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER", "trust": false, "fail_on_drop": true, "port": "5432"}}, "trust": false, "db": "ep2_alfa_bank", "changed": true}\r\n', 'Shared connection to myhost closed.\r\n')
...
myhost                  : ok=2    changed=1    unreachable=0    failed=0    skipped=0
```
Nice job.